accessibility/accessibility.md

```diff
@@ -56,19 +60,22 @@ Myth: We only need to consider color contrast when designing for vision impairme
 Fact: We also need to consider text size, line height, font,
 
 ### Independent Practice
-1. Read the [Accessibility section on the Google Developers Site](https://developers.google.com/web/fundamentals/accessibility/) through the "Accessible Styles" chapter.
+
+1. Read the [Accessibility section](https://developers.google.com/web/fundamentals/accessibility/) on the Google Developers Site through the "Accessible Styles" chapter.
 2. Download the [aXe Chrome Extension.](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
 2. Follow the guide on [How to Do an Accessibility Review](https://developers.google.com/web/fundamentals/accessibility/how-to-review)
-- What are the main user goals/paths in your Portfolio Page?
-- Are they accessible?
-- How can you make them accessible?
-- Write down three specific things you can do to make your Portfolio Page more accessible.
 
-### Additional resources
+### Supplemental Resources
 
-- [Intro to web accessibility](https://www.creativebloq.com/netmag/simple-introduction-web-accessibility-7116888)
+- [Introduction to Web Accessibility](https://www.creativebloq.com/netmag/simple-introduction-web-accessibility-7116888)
 - [Web experiences of people with disabilities](https://webaim.org/intro/#people)
 - [Google accessibility course (text)](https://developers.google.com/web/fundamentals/accessibility/) 
 - [Google accessibility course (video)](https://www.udacity.com/course/web-accessibility--ud891)
 - [A11Y Style Guide](https://a11y-style-guide.com/style-guide/)
 
+### Check for Understanding
+
```

Comment:
Are you solving a check for understanding issue or a materials issue?

Reply:
No. But for this file the questions where already there in Guided Practice so I shifted them here and also added one question

Reply:
ah, thank you.